test/TestImportTasks.py

```diff
@@ -0,0 +1,73 @@
+"""Calling 'import_task' from multiple files in different directories."""
+import os
+import shutil
+import tempfile
+import textwrap
+
+import pytest
+
+from ansiblelint.runner import Runner
+
+LAYOUT = {
+    'directories': ['tasks', 'tasks/subtasks'],
+    'files': {
+        'main.yml': textwrap.dedent(
+            """\
+            ---
+            - hosts: target
+              gather_facts: false
+              tasks:
+                - name: from main import task 1
+                  import_tasks: tasks/task_1.yml
+            """
+        ),
+        'tasks/task_1.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from task 1 import task 2
+              import_tasks: tasks/task_2.yml
+            """
+        ),
+        'tasks/task_2.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from task 2 import subtask 1
+              import_tasks: tasks/subtasks/subtask_1.yml
+            """
+        ),
+        'tasks/subtasks/subtask_1.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from subtask 1 import subtask 2
+              import_tasks: tasks/subtasks/subtask_2.yml
+            """
+        ),
+        'tasks/subtasks/subtask_2.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from subtask 2 do something
+              debug:
+                msg: |
+                  Something...
+            """
+        ),
+    },
+}
+
+
+@pytest.mark.xfail
+def test_import_tasks(default_rules_collection):
+    """Calling 'import_task' from multiple files in different directories."""
+    root_dir_path = tempfile.mkdtemp()
+
+    for directory in LAYOUT['directories']:
+        os.makedirs(os.path.join(root_dir_path, directory), exist_ok=True)
```

Comment:
Using `tempfile` leaves garbage on disk in case of failures. Please use the built-in `tmp_path` fixture that pytest provides instead: https://docs.pytest.org/en/stable/tmpdir.html#the-tmp-path-fixture.

Also, that fixture provides a `pathlib.Path` object so you could use a cleaner interface of `pathlib.Path.write_text()` to save files on disk.